articles/connections/database/custom-db/templates/get-user.md

```diff
@@ -56,141 +56,240 @@ Auth0 provides sample scripts for use with the following languages/technologies:
 
-  callback(null, profile);
+  client.connect(function (err) {
+    if (err) return callback(err);
+
+    const db = client.db('db-name');
+    const users = db.collection('users');
+
+    users.findOne({ email: email }, function (err, user) {
+      client.close();
+
+      if (err) return callback(err);
+      if (!user) return callback(null, null);
+
+      return callback(null, {
+        user_id: user._id.toString(),
+        nickname: user.nickname,
+        email: user.email
+      });
+    });
+  });
 }
 ```
 
 ### MySQL
 
 ```
-function getByEmail (name, callback) {
-  var profile = {
-    user_id:     "103547991597142817347",
-    nickname:    "johnfoo",
-    email:       "johnfoo@gmail.com",
-    name:        "John Foo",
-    given_name:  "John",
-    family_name: "Foo"
-  };
+function getByEmail(email, callback) {
+  const mysql = require('mysql');
 
-  callback(null, profile);
+  const connection = mysql({
+    host: 'localhost',
+    user: 'me',
+    password: 'secret',
+    database: 'mydb'
+  });
+
+  connection.connect();
+
+  const query = 'SELECT id, nickname, email FROM users WHERE email = ?';
+
+  connection.query(query, [ email ], function(err, results) {
+    if (err || results.length === 0) return callback(err || null);
+
+    const user = results[0];
+    callback(null, {
+      user_id: user.id.toString(),
+      nickname: user.nickname,
+      email: user.email
+    });
+  });
 }
 ```
 
 ### Oracle
 
 ```
 function loginByEmail(email, callback) {
-  var oracledb = require('oracledb');
+  const oracledb = require('oracledb');
   oracledb.outFormat = oracledb.OBJECT;
 
   oracledb.getConnection({
       user: configuration.dbUser,
       password: configuration.dbUserPassword,
-      connectString: "CONNECTION_STRING" // Refer here https://github.com/oracle/node-oracledb/blob/master/doc/api.md#connectionstrings
+      connectString: 'CONNECTION_STRING' // Refer here https://github.com/oracle/node-oracledb/blob/master/doc/api.md#connectionstrings
     },
     function(err, connection) {
-      if (err) {
-        return callback(err);
-      }
-      connection.execute(
-        "SELECT ID, EMAIL, PASSWORD, EMAIL_VERIFIED, NICKNAME" +
-        " FROM Users " +
-        " WHERE EMAIL = :email", [email],
-        function(err, result) {
-          if (err) {
-            doRelease(connection);
-            return callback(new Error(err));
-          }
-          doRelease(connection);
-          if (result.rows.length === 0) {
-            return callback(null);
-          }
-          var userProfile = {
-            user_id: result.rows[0].ID,
-            nickname: result.rows[0].NICKNAME,
-            email: result.rows[0].EMAIL,
-            email_verified: result.rows[0].EMAIL_VERIFIED
-          };
-          callback(null, userProfile);
-        });
+      if (err) return callback(err);
+
+      const query = 'select ID, EMAIL, NICKNAME from Users where EMAIL = :email';
```

Comment:
In Heroku review app, code is not rendered correctly from here on (green, when should not be):
https://docs-content-staging-pr-8876.herokuapp.com/docs/connections/database/custom-db/templates/login#oracle